packages/relay/src/lib/db/repositories/hbarLimiter/ipAddressHbarSpendingPlanRepository.ts

```diff
@@ -102,6 +102,36 @@ export class IPAddressHbarSpendingPlanRepository {
     }
   }
 
+  /**
+   * Deletes all IP address spending plans from the cache.
+   *
+   * @param {RequestDetails} requestDetails - The request details for logging and tracking.
+   * @returns {Promise<void>} - A promise that resolves when all IP address spending plans are deleted.
+   */
+  async deleteAll(requestDetails: RequestDetails): Promise<void> {
```

Comment:
To keep this class, `IPAddressHbarSpendingPlanRepository`, and the `EthAddressHbarSpendingPlanRepository` class in sync, let's also add another deleteAll() method to the `EthAddressHbarSpendingPlanRepository` class.